.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contribution Guidelines
+
+## Reporting Issues
+
+Please [open an issue](https://github.com/serokell/tezos-client/issues/new)
+if you find a bug or have a feature request.
+Before submitting a bug report or feature request, check to make sure it hasn't already been submitted
+
+The more detailed your report is, the faster it can be resolved.
+If you report a bug, please provide steps to reproduce this bug and revision of code in which this bug reproduces.
+
+
+## Code
+
+If you would like to contribute code to fix a bug, add a new feature, or
+otherwise improve our project, pull requests are most welcome.
+
+Our pull request template contains a [checklist](./pull_request_template.md#white_check_mark-checklist-for-your-pull-request) of acceptance criteria for your pull request.
+Please read it before you start contributing and make sure your contributions adhere to this checklist.
```

Comment:
Since there is only 1 item, maybe this paragraph doesn't make much sense. It is useful when there are checkmarks about tests, because it's better to keep in mind that you add tests before you make a PR. In our case checklist is quite trivial.